Public/Src/Engine/Dll/HistoricTableSizes.cs

```diff
@@ -199,7 +199,7 @@ public override bool Equals(object obj)
         /// <inheritdoc/>
         public override int GetHashCode()
         {
-            return HashCodeHelper.Combine(Count, SizeInBytes);
+            return HashCodeHelper.Combine(Count, (int)(SizeInBytes & 0xFFFFFFFF), (int) (SizeInBytes >> 32));

```

Comment:
this is dirty hacking - i'd appreciate any input here

Reply:
Another alternavie would be using bitconverter to create a bytearray to be add to hascode helper. But I like this pattern better way more effecient.

Allthough perhaps this belongs inside the hasCodehelper itself.


---
In reply to: [291774166](https://github.com/microsoft/BuildXL/pull/432#discussion_r291774166) [](ancestors = 291774166)